test/test_from_sdist.py

```diff
@@ -61,10 +61,23 @@ def test_simple(tmp_path):
     sdist_dir.mkdir()
     sdist_path = make_sdist(basic_project, sdist_dir)
 
+    setup_py_assertion_snippet = textwrap.dedent(
+        """
+        import os
+
+        assert os.path.exists("setup.py")
+        assert os.path.exists("{package}/setup.py")
```

Comment:
Try this additionally:
```suggestion
        assert os.path.exists('setup.py')
        assert os.path.exists('{package}/setup.py')
```